server.js

```diff
@@ -3,21 +3,85 @@ const cors = require("cors");
 
 const app = express();
 
-app.use(cors());
+const bodyParser = require("body-parser");
+app.use(bodyParser.urlencoded({ extended: true }));
 
-const welcomeMessage = {
+app.use(cors());
```

Comment:
Let me know if you want a refresher on CORS? Can't remember how much we talked about it.